test/runtests.jl

```diff
@@ -18,6 +18,12 @@ const MOI = MathOptInterface
 const MOIT = MOI.Test
 const MOIU = MOI.Utilities
 
+# Test that the macro call `m` throws an error exception during pre-compilation
+macro test_macro_throws(m)
```

Comment:
To be consistent with `@test_throws`, this could be `@test_macro_throws(errortype, ...)` or `@test_macro_throws_error`.